pkg/v22/resource/clusterconfigmap/desired.go

```diff
@@ -46,26 +45,55 @@ func (r *StateGetter) GetDesiredState(ctx context.Context, obj interface{}) ([]*
 		}
 	}
 
-	values := map[string]interface{}{
-		"baseDomain":   key.TenantBaseDomain(clusterConfig),
-		"clusterDNSIP": clusterDNSIP,
-		"clusterID":    key.ClusterID(clusterConfig),
-		"ingressController": map[string]interface{}{
-			"replicas": ingressControllerReplicas,
+	configMapSpecs := []configMapSpec{
+		{
+			Name:      key.ClusterConfigMapName(clusterConfig),
+			Namespace: key.ClusterID(clusterConfig),
+			Values: map[string]interface{}{
+				"baseDomain":   key.TenantBaseDomain(clusterConfig),
+				"clusterDNSIP": clusterDNSIP,
+				"clusterID":    key.ClusterID(clusterConfig),
+			},
+		},
+		{
+			Name:      key.IngressControllerConfigMapName,
+			Namespace: key.ClusterID(clusterConfig),
+			Values: map[string]interface{}{
+				"baseDomain": key.TenantBaseDomain(clusterConfig),
+				"clusterID":  key.ClusterID(clusterConfig),
+				"ingressController": map[string]interface{}{
+					"replicas": ingressControllerReplicas,
+				},
```

Comment:
We need all these values not just the replicas. See https://github.com/giantswarm/nginx-ingress-controller-app/blob/0264ae24f53f5530bd07511d94ec95531d732a35/helm/nginx-ingress-controller-app/values.yaml#L9L21.